src/others/components/map/CircleLayerStyle.tsx

```diff
@@ -1,7 +1,7 @@
 import { LayerProps } from "react-map-gl";
 
 export const layerStyle: LayerProps = {
-  id: "ukr_water_needs-point",
+  id: "",
```

Comment:
Do we need to specify `id` at all?

Per the docs,
```
Unique identifier of the layer. If not provided, a default id will be assigned.
```

Reply:
I don't think so - but I don't know how much of the UI is determined by the set up in https://cloud.maptiler.com/maps/.  
Maybe @AntonRaichuk could make us a short demo on that?

Reply:
I'm not sure demo of what you are asking for.

I'm a bit confused why ids are removed, but if it works without them - sure.
A potential problem is that we'll start adding more sources/layers soon, so I'm not sure if without ids it will still be working properly.

Reply:
Getting this in count maybe we could mark this change as obsolete then?

Reply:
Since the error is annoying and the id is definitely not "ukr_water_needs-point", which I find confusing - so either let's agree on what the id represents and set it properly everywhere, or remove it altogether.

Reply:
ID is "ukr_water_needs-point", it just appears later.
Error only occurs if you start using your mouse before the map is fully loaded. 

If you don't touch anything for a bit and then start moving everything works as expected.

Reply:
Now we have 2 interactive layers. Does this fix still works at head?